Emby.Naming/Video/VideoListResolver.cs

```diff
@@ -141,7 +141,13 @@ private static List<VideoInfo> GetVideosGroupedByVersion(List<VideoInfo> videos,
                 {
                     if (group.Key)
                     {
-                        videos.InsertRange(0, group.OrderByDescending(x => x.Files[0].FileNameWithoutExtension.ToString(), new AlphanumericComparator()));
+                        videos.InsertRange(0, group
+                            .OrderByDescending(x => ResolutionRegex().Match(x.Files[0].FileNameWithoutExtension.ToString()).Value, new AlphanumericComparator())
+                            .ThenBy(x =>
+                            {
+                                var resolutionMatch = ResolutionRegex().Match(x.Files[0].FileNameWithoutExtension.ToString());
+                                return x.Files[0].FileNameWithoutExtension.ToString()[(resolutionMatch.Index + resolutionMatch.Length)..];
+                            }));
```

Comment:
I thought about:
```suggestion
                            .ThenBy(x => x.Files[0].FileNameWithoutExtension.ToString(), new AlphanumericComparator()));
```
It's simpler, but will probably look like:
```
... - 2160p
... - 2160p B
... - A 2160p
... - B 2160p
... - 1080p B
... - A 1080p
... - B 1080p
```
🤷‍♂️

Reply:
I think it is OK to simplify it this way as either way the mixed versioning will have some problem that may look less natural. Just don't think about that too much for now